xapian-applications/omega/mimemap.tokens

```diff
@@ -199,6 +199,20 @@ lrf		application/x-sony-bbeb
 tcr		application/x-tcr-ebook
 pdb		application/vnd.palm
 
+# Image files:
+gif		image/gif
+jfif		image/jpeg
+jpe		image/jpeg
+jpeg		image/jpeg
+jpg		image/jpeg
+pbm		image/x-pbm
+pgm		image/x-pgm
```

Comment:
libmagic seems to give `image/x-portable-pixmap` and `image/x-portable-greymap` (and that's what https://en.wikipedia.org/wiki/Netpbm_format lists too).  It makes sense to align with libmagic since then we'll correctly handle a file which doesn't have an extension listed here but which libmagic identifies (for example `.pnm` means "one of PBM, PGM or PPM").

Also, perhaps we should have an entry here for `pnm`, since we just pass all these formats to tesseract and don't actually care which of them it is internally.

Reply:
Actually, I run `file --mime-type` and it returns `application/octet-stream` for these files but I am going to change the mimetypes. It is a good idea(and actually necessary) to get some kind of compatibility with libmagic.

Reply:
What `file --mime-type` says will depend on which version of `file(1)` you're running. `application/octet-stream` just means "it's not text but we don't know what it is".

Reply:
For me, running Debian unstable (which should have a recent version of `file`/libmagic) and looking at all the files with one of these extensions on my machine:

```
$ locate -0 '*.p[bgnp]m' | xargs -0 file --mime-type | awk '{print $2}' | sort | uniq -c
      2 application/octet-stream
      1 application/x-pie-executable
     52 image/x-portable-bitmap
     91 image/x-portable-greymap
    108 image/x-portable-pixmap
      1 inode/x-empty
      1 text/plain
```

The `application/x-pie-executable` is `/usr/lib/grass76/bin/r.out.ppm`, which seems to be a helper program to export PPMs from grass (a GIS), not an image file at all.

The `inode/x-empty` is an empty temporary file something left behind.

The `text/plain` and one `application/octet-stream` are deliberately broken example image files in a test suite in some source code I have lying around.

So for me, all the files on my machine are reported as one of the types, except one - looking at the contents that seems to have the end of the file padded with a load of zero bytes.  It displays OK in an image viewer though.